src/testSetup.ts

```diff
@@ -975,6 +976,7 @@ export class MockTestOrgData {
     this.accessToken = `${this.testId}/accessToken`;
     this.refreshToken = `${this.testId}/refreshToken`;
     this.redirectUri = 'http://localhost:1717/OauthRedirect';
+    this.password = Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
```

Comment:
It makes sense to have all the "always there" properties on the Mock, since you're going to need them anyway.

Would anyone ever want a MockTestOrgData without a password (ex: to UT the output of `org:display`)?  

If so, they would do this (remove it from the mock after it's created)
https://github.com/forcedotcom/sfdx-core/blob/7a8ff9aebb19226b2d3df8c5b4b008cf9aaca38e/test/unit/org/authInfoTest.ts

instead of adding it to the mock if the need it via the public password prop?